### DISCUSSION TITLE
How to represent networkx graphs over time-series?

### DISCUSSION THREAD

**Discussion by adam2392** [networkx/networkx#5525] [Q&A] [answered]
Is there a recommended way to create networkx graphs over time that is efficient, not super verbose and leverages the existing algorithms in networkx?

**Comment by dschult** [comment, marked as answer]
We don't have many algorithm for time-varying graphs. As far as I can tell, the algorithms haven't evolved and stabilized in this setting. The NetworkX Graph structure is quite flexible and can store the temporal information as node and edge attributes. But the best way to do that varies with the data you have and the questions you are trying to answer. 

You might take a look at #3548
